### PR TITLE
Clarify developer installation instructions, and fix up some metadata.

### PR DESCRIPTION
**Pull request by ceball**
* Clarify developer installation instructions.
* Fix nodejs dependency (I don't _think_ it is required to run the tests...right? It is required for developers, but they are instructed to install the build extra, which includes nodejs.)

The changes to the developer guide should definitely be reviewed. Also, I've never created or edited a panel model (as far as I can recall!), so I'm sure someone could add something more useful...

Also, because I was in there:
  * Update license years and team name.
  * Update package metadata.
  * ~Removed `postBuild` (am I right it's unused?)~

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -98,7 +98,6 @@ jobs:
         - source activate test-environment
         - doit develop_install $CHANS_DEV -o build && pip uninstall -y panel
         - doit pip_on_conda
-        - conda install $CHANS_DEV nodejs
```
Comment (ceball) [inline .travis.yml]: The develop_install command above will install it (`-o build`).

File: LICENSE.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2018, PyViz contributors (pyviz.org)
+Copyright (c) 2018-2020, HoloViz team (holoviz.org).
```
Comment (ceball) [inline LICENSE.txt]: By team I mean you 😂

Comment (philippjfr) [reply LICENSE.txt]: 😄

**Comment by philippjfr** [comment]
The postBuild should have a note in it to mention that it's used for binder. It is indeed still needed for that reason.

**Comment by philippjfr** [comment]
Thanks, looks good to me!

**Pull request merged**